### PR TITLE
Log LED states on every call to syncLeds()

### PR DESCRIPTION
**Pull request by cdisselkoen**
Posting this as a PR mostly to get feedback - if everyone (and in particular @noseglasses) thinks it's ok and Travis passes, I'll go ahead and merge this.

### CODE REVIEW

**Comment by noseglasses** [comment]
Looks great!

**Comment by noseglasses** [comment]
One tiny thing. Could you add a getter for row/col indexed access?

File: src/Kaleidoscope-Hardware-Virtual.h [hunk 1]
```diff
@@ -53,12 +53,12 @@ class Virtual {
...
-  void setCrgbAt(byte /*row*/, byte /*col*/, cRGB /*color*/) {}
-  void setCrgbAt(uint8_t /*i*/, cRGB /*color*/) {}
-  cRGB getCrgbAt(uint8_t /*i*/) {
-    return CRGB(0, 0, 0);
-  }
+  // For virtual hardware, the current state of all LEDs will be logged to a dedicated file in results upon each call to syncLeds()
+  void syncLeds(void);
+  void setCrgbAt(byte /*row*/, byte /*col*/, cRGB /*color*/);
+  void setCrgbAt(uint8_t /*i*/, cRGB /*color*/);
+  cRGB getCrgbAt(uint8_t /*i*/);
```
Comment (noseglasses) [inline src/Kaleidoscope-Hardware-Virtual.h]: We should respect const-correctness for getters. Should be `cRGB getCrgbAt(...) const;`.

Comment (cdisselkoen) [reply src/Kaleidoscope-Hardware-Virtual.h]: Good catch.

**Comment by cdisselkoen** [comment]
Yeah, I can add a getter that works by row/col.  That's interesting because it's not part of the Kaleidoscope-Hardware API which we're nominally implementing here.  But there's no reason we can't have extra methods not part of the official API (effectively subclassing Kaleidoscope-Hardware).

**Pull request merged**